modules/dictionary/php/fields.class.inc

```diff
@@ -0,0 +1,132 @@
+<?php declare(strict_types=1);
+namespace LORIS\dictionary;
+
+use \Psr\Http\Message\ServerRequestInterface;
+use \Psr\Http\Message\ResponseInterface;
+/**
+ * The dictionary/fields endpoint handles requests to update a specific
+ * field description
+ *
+ * @license http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ */
+
+class Fields extends \NDB_Page
```

Comment:
no need for NDB_Page. `implements RequestHandlerInetrface` would be enough.

Reply:
I don't see what the benefit of that would be.

Reply:
Better cohesion and reduced coupling. If you only need the handle function, and maybe _hasAccess, implementing  RequestHandlerInterface is enough. I don't see what $identifer and $commentid means for a datadictionnary field. $fieldoptions, $dateoption adds confusion. What is the use of AddFile, AddSelect, ...? Will it use its LorisForm?

Reply:
To be honest I'd like to remove `$identifier` and `$commentid` from the page class in general, but when I tried doing that in #7414 the scope of the change exploded so I held back to do it later.

I don't think having methods that are uncalled adds any overhead and the properties in the page class add minimal. I just had a look through the code and hasAccess does seem to be the main benefit. Other than that, I think there's benefit in having a root to the class hierarchy, but I can change this to manually call hasAccess if you want.

Reply:
I made #7431 to work through what would be ideal in general outside of this PR.